### ISSUE TITLE
Applying profile based on ID reports Profile Not Found

### ISSUE THREAD

**Issue by Stwerp** [Prayag2/konsave#77]
** Description**
This may be intended behavior *BUT* it does not match use-case of the `gif` in the readme/github landing page.

`konsave -l` produces a list such as:
```
Konsave profiles:
ID            NAME
1              ProfileA
2              ProfileB
```

Attempting `konsave -a 2` produces
```
Konsave: Profile not found :(
Please check the log at <$HOME>/.cache/konsave_log.txt for more details
```

It's unclear what the ID is used for. The landing page video shows the user typing `konsave --apply 2` to apply a new profile.

I just installed konsave from pip today and its version `2.2.0`

**System information**
 - running Rocky Linux 8 (red hat clone)
 - How did you install Konsave? using pip
 - What version of Konsave are you using? version 2.2.0
 - Did you try updating? no, but 2.2.0 looks like latest release per github sidebar

Again, this could be intended behavior, and then the issue is that the video is incorrect. Or I've messed something up on my end :) It's a new laptop I'm trying to get up and going.

**Comment by Prayag2** [comment]
> This may be intended behavior

Yes, this indeed is the intended behavior. IDs were used in older versions and are now obsolete. I'll update the gif in the README and remove IDs from the output in the next release.

**Comment by ben2talk** [comment]
Would it not be better, as is the case with 'yay' and 'paru' wrappers, to be able to (without recalling all of our profiles) simply do 'konsave -a' or 'konsave -r' or even 'konsave -s' and be offered a list, with the option to enter the ID to apply?

**Comment by Bekon241** [comment]
I install this today i save profile and ksvm  then import it and i want apply konsave -a and i cant :( becouse error.

Konsave: Profile not found :(
Please check the log at <$HOME>/.cache/konsave_log.txt for more details
This is problem for me because i setup my OS perfect for me and i want import it and active to another pc...

I create simple name but my config file is on hidden mounted drive, but i try also from home dir and still same problem... Maybe i cant choice profile to active if i have only one?

In Nobara (fedora 43) KDE


Ok i know where is problem when i use flag example -a not full like --apply then i have this problem i delete all profiles i create new only in home directory and i use full flags then no error anymore. Also with short version too :) This was soo wierd but all work for me now i dont know what i do bad before but all work :)

For all who need clean steps.
first --save profile
check on list konsave --list
if you see it on list --import-profile
then --apply-profile
Logout and login back.

I not test this yet on other pc but i do this soon. I rly dont know why i have error before... Maybe too long name file?
